README.rst

```diff
@@ -52,38 +52,90 @@ What is PUDL?
 -------------
 
 The `PUDL <https://catalyst.coop/pudl/>`__ Project is an open source data processing
-pipeline that makes US energy data easier to access and use programmatically.
+pipeline created by `Catalyst Cooperative
+<https://catalyst.coop/>`__ that cleans, integrates, and standardizes some of the most
+widely used public energy datasets in the US. Hundreds of gigabytes of valuable data
+are published by US government agencies, but they are often difficult to work with.
+PUDL takes the original spreadsheets, CSV files, and databases and turns them into a
+unified resource.
+
+PUDL is comprised of three core components:
+
+- **Raw Data Archives**
+
+  - We `archive <https://github.com/catalyst-cooperative/pudl-archiver>`__ all the raw
+    data inputs on `Zenodo <https://zenodo.org/communities/catalyst-cooperative/?page=1&size=20>`__
+    to ensure perminant, versioned access to the data. In the event that an agency
+    changes how they publish data or deletes old files, the ETL will still have access
+    to the original inputs. Each of the data inputs may have several different versions
+    archived, and all are assigned a unique DOI and made available through the REST API.
+- **ETL Pipeline**
+
+  - The ETL pipeline (this repo) ingests the raw archives, cleans them, integrates
+    them, and outputs them to a series of tables stored in SQLite Databases, Parquet
+    files, and pickle files (the Data Warehouse). Each release of the PUDL Python
+    package is embedded with a set of of DOIs to indicate which version of the raw
+    inputs it is meant to process. This process helps ensure that the ETL and it's
+    outputs are replicable.
+- **Data Warehouse**
+
+  - The outputs from the ETL, sometimes called "PUDL outputs", are stored in a data
+    warehouse so that users can access the data without having to run any code. The
+    majority of the outputs are stored in ``pudl.sqlite``, however CEMS data are stored
+    in seperate Parquet files due to their large size. The warehouse also contains
+    pickled interim assets from the ETL process, should users want to access the data
+    at various stages of the cleaning process, and SQLite databases for the raw FERC
+    inputs.
+
+For more information about each of the components, read our
+`documentation <https://catalystcoop-pudl--2874.org.readthedocs.build/en/2874/intro.html>`__
+.
```

Comment:
I think including this early in the readme forces users to scroll through more text to get to the data access section which I'm assuming is what they care about.

I think this type of architecture information is more important for contributors which will be reading through the Development section of the docs.

Reply:
That's a fair point. I don't want to assume users know what they want yet though and this provides them with the opportunity to understand what happens to the data before they use it. Maybe we could run this by some other people to see what they think.

Reply:
We could also just copy what's in the intro and use that instead: 

```
- **Raw Data Archives** (raw, versioned inputs)
- **ETL Pipeline** (code to process, clean, and organize the raw inputs)
- **Data Warehouse** (location where ETL outputs, both interim and final, are stored)
```